### PR TITLE
Fixes #23978: Prompt warning and ask confirmation for factory reset

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/23978

### CODE REVIEW

File: share/commands/agent-factory-reset [hunk 1]
```diff
@@ -62,6 +62,17 @@ while getopts "iIvdqcTf" opt; do
   esac
 done
 
+while true; do
+    echo "This command will permanently delete your node uuid and keys"
```
Comment (ElaadF) [inline share/commands/agent-factory-reset:66]: We need a real message, this is kind of a placeholder, can you suggest a message ?

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by ElaadF** [comment]
OK, squash merging this PR

**Pull request merged**